doc/api/stream.md

```diff
@@ -1667,6 +1667,13 @@ of a stream that are intended for use by consumers (as described in the
 [API for Stream Consumers][] section). Doing so may lead to adverse side effects
 in application code consuming the stream.
 
+It is highly discouraged to override any public method or to emit any internal
```

Comment:
List which methods just like events? And how about prepend a "parent" on the "public method"?

Reply:
What do you mean by "parent"?

Reply:
Added method examples.